File: bubble_sort.js
// bubble sort by javascript
var list =[3,7,4,6,4,9,8,23,11,33,21,13]
function bubbleSort(list){
    for(var i = 0 ; i < list.length; i++){
        for(var j = 0; j < list.length - i - 1 ;j++ ){
            if(list[j] > list[j+1]){
                T = list[j]
                list[j] = list[j+1]
                list[j+1] = T
            }
        }
    }
    return list
}
console.log(bubbleSort(list))

File: bubble_sort.py
# bubble sort by python
array=[5,7,4,3,8,5,9,43,46,75,54,756,54]
def Bubble_sort(arr):
    for i in range(len(arr)):
        for j in range(len(arr)-i-1):
            if(arr[j]>arr[j+1]):
                T = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = T
    return  arr

print(Bubble_sort(array))    
